apigo/api/preview.go

```diff
@@ -31,8 +31,8 @@ func init() {
 
 // PreviewRequest is the body of the POST /preview request.
 type PreviewRequest struct {
-	ResourceID       string   `json:"resource_id"`
-	PrimaryKeyValues []string `json:"primary_key_values"`
+	ResourceID       string        `json:"resource_id"`
+	PrimaryKeyValues []json.Number `json:"primary_key_values"`
```

Comment:
Does it also work with strings that don’t represent numbers?

Reply:
it does not. see comment above and updated version